apps/docs/pages/guides/getting-started/tutorials/with-flutter.mdx

```diff
@@ -5,6 +5,15 @@ export const meta = {
   description: 'Learn how to use Supabase in your Flutter App.',
```

Comment:
@dshukertjr we're starting to move videos to the TOC section for better screen real estate:
![image](https://github.com/supabase/supabase/assets/4133076/ca3e91c4-520d-4117-9bde-3020494c06e1)

which can be done using the `tocVideo` key in `meta`:
```suggestion
  description: 'Learn how to use Supabase in your Flutter App.',
  tocVideo: 'r7ysVtZ5Row',
```